Lib/test/test_ctypes/test_wintypes.py

```diff
@@ -38,6 +41,19 @@ def test_variant_bool(self):
         vb.value = []
         self.assertIs(vb.value, False)
 
+    def assertIsSigned(self, ctype):
+        self.assertLess(ctype(-1).value, 0)
+
+    def test_signedness(self):
+        for type in (wintypes.BYTE, wintypes.WORD, wintypes.DWORD,
+                     wintypes.BOOLEAN, wintypes.UINT, wintypes.ULONG):
+            with self.subTest(type=type):
+                self.assertFalse(assertIsSigned(type))
+
+        for type in (wintypes.BOOL, wintypes.INT, wintypes.LONG):
+            with self.subTest(type=type):
+                self.assertTrue(assertIsSigned(type))
```

Comment:
This should be

```python
    self.assertIsSigned(type)
```

This is Python, not C++ - implicit "this" is not a feature. :)

Reply:
Thank you, fixed again. At least, `python -m test test_ctypes.test_wintypes` gives `Tests result: SUCCESS`.

It seems to be a bad idea to build a lean Qt 5 while working on the PR :D (I threw in a bunch of `--no-feature-*` into its `configure` and now meticulously remove them one by one after each hour-long build attempt because Qt components need most of them and fail the build, reporting a single missing feature each time).

So my apologises for such mistakes. I hope I've made no extra mistake.